### ISSUE TITLE
allow environment variables in connection properties

### ISSUE THREAD

**Issue by sethishi** [chop-dbhi/prometheus-sql#44]
We would like to use environment variables we can inject into the different instances of the prometheus-sql agents to ensure that we can run a docker container per environment we want to monitor.
We could not find anything so far, and when we tried to add the environment variable in our queries.yml file, it cannot connect and seems not to resolve the host:
`[prometheus_query] 2018/04/23 15:09:16 503 Service Unavailable: problem connecting to database: dial tcp: lookup ${MYSQL_HOST}: no such host
`
Basically what we want to do is add the following block. We currently tried it in the queries.yml file, but would at some point like to use these variables in the config file to prevent duplication.
```
connection:
        host: ${MYSQL_HOST}
        port: 3306
        user: ${MYSQL_USER}
        password: ${MYSQL_PASSWORD}
        database: ${MYSQL_DB}
```

Do you already support this behaviour? If not is there a workaround for now, or a plan to support that feature?

Thanks

**Comment by bruth** [comment]
Good suggestion! This is straightforward to add.

**Comment by bruth** [comment]
Opened a PR in #45
